src/selectors/getAllProjectKeys.js

```diff
@@ -1,10 +1,26 @@
 import {createSelector} from 'reselect';
+import isNull from 'lodash-es/isNull';
 
 export default createSelector(
   state => state.get('projects'),
   projects => projects.
-    filter(project => project.updatedAt).
-    sortBy(project => -project.updatedAt).
+    sort((...args) => {
+      const [firstProj, secondProj] = args;
```

Comment:
My preference is to just spell full words out unless there is a _really_ good reason not to (so `Project` here and below)

Reply:
:( and I thought i was being concise with the variable naming.